dev_server/dev_server.py

```diff
@@ -173,8 +174,8 @@ def main(options):
 
     try:
         index = imp.load_source('index', 'index.wsgi')
-    except IOError:
```

Comment:
some IOError exception may be thrown during the import process, but that doesn't mean 'index.wsgi' doesn't exist.
directly print the traceback can help user figure out where the real bug is.